src/services/identity/local.ts

```diff
@@ -122,11 +122,17 @@ export class LocalIdentityService extends DefaultIdentityService {
 		try {
 			return await this.getDid(ISSUER_DID);
 		} catch {
+			const key = {
+				kid: ISSUER_PUBLIC_KEY_HEX,
+				type: 'Ed25519' as TKeyType,
```

Comment:
Switch to the following for consistency:

```suggestion
				type: 'Ed25519' as const,
```